pkg/elle/core/core.go

```diff
@@ -24,7 +36,7 @@ func MergeAnomalies(anomalies ...Anomalies) Anomalies {
 type DataExplainer interface {
 	// Given a pair of operations a and b, explains why b depends on a, in the
 	//    form of a data structure. Returns `nil` if b does not depend on a.
-	ExplainPairData() interface{}
+	ExplainPairData(p1, p2 PathType) ExplainResult
```

Comment:
PathType seems always be core.Op type? IMO we can use core.Op here directly.

Reply:
OK